backend/codesirius/codesirius/aws_client.py

```
@@ -0,0 +1,29 @@
+from threading import Lock
+from typing import Dict, Type
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)